docs/en/reference/annotations-reference.rst

```diff
@@ -715,6 +717,7 @@ Required attributes:
 
 -  **targetEntity**: FQCN of the referenced target entity. Can be the
    unqualified class name if both classes are in the same namespace.
+   When typed properties are used it is inherited from PHP type.
```

Comment:
I would formulate this a bit differently: `You can omit this value if you use a PHP property type instead.`

Reply:
OK, changed